frontend/test/e2e/caselaw/legal-periodical-evaluation.spec.ts

```diff
@@ -156,13 +143,33 @@ test.describe(
             .click()
         })
 
-        await test.step("prefix or name have to be chosen", async () => {
-          await page.getByLabel("Auswertung starten").click()
+        await test.step("A new evaluation is started using the “Neue Periodikaauswertung” button.", async () => {
+          await expect(
+            page.getByLabel("Neue Periodikaauswertung"),
+          ).toBeVisible()
+          await page.getByLabel("Neue Periodikaauswertung").click()
+          await expect(page).toHaveURL(
+            /\/caselaw\/periodical-evaluation\/[0-9a-fA-F\-]{36}\/edition/,
+          )
+        })
+
+        await test.step("The inputs are correctly validated (prefix or name have to be chosen, suffix is optional)", async () => {
+          await expect(page.getByText("Pflichtfeld nicht befüllt")).toBeHidden()
+          // TODO should be empty from the beginning
+          await clearInput(page, "Name der Ausgabe")
+          await page.getByLabel("Speichern").click()
           await expect(
             page.locator(`text="Pflichtfeld nicht befüllt"`),
           ).toHaveCount(1)
         })
 
+        // await test.step("prefix or name have to be chosen", async () => {
```

Comment:
This line can be removed as it deprecated to our current validation

Reply:
Thank you! Done ✅

Reply:
Done ✅